src/tokens/config.js

```diff
@@ -41,6 +41,11 @@ module.exports = {
             showFileHeader: true,
           },
         },
+        {
+          destination: 'index.ts',
```

Comment:
Maybe this should output a JS file, to be more easily consumable in other apps? As it's just an object, I don't think it'll make any difference to typings

Reply:
I'm not super opposed to `css/index.js`. Maybe it could be `styles/index.js` and `styles/variables.css` but I don't have strong feelings.

Reply:
Also don't have terribly strong feelings, but lean towards `styles `.

However as I was writing that, I may have convinced myself otherwise because though this is not css, it requires the CSS variables, and therefore isn't really it's own thing. Just a different way of accessing the same thing.

Admittedly an argument could be made that this is also true of Tailwind, but there are other notable differences there like how the variables are built for RGB.

Reply:
Calling it `styles` seems a bit generic to me, especially since we have `/ios`, `/android`, `/skia` (which maybe should be prefixed cpp as skia has some non-c++ implementations AFIAK), etc... "/styles" could be relevant to any of those languages, whereas this JS object and the css vars can only be used in a platform supporting CSS (e.g. not react native JS either). So to keep with the naming convention of using language or framework, doesn't keeping `/css` seem appropriate?

Reply:
> Maybe this should output a JS file,

For some reason I'm not getting intellisense on the object in brave-core when it's a JS file. Will keep experimenting.

Reply:
> So to keep with the naming convention of using language or framework, doesn't keeping /css seem appropriate?

I think I've come around and would agree that `css/` is the write place.

Reply:
I don't have strong enough feelings on it to suggest either way. `css` or `styles` is fine :smile:

Reply:
Let's just stick with CSS.

Reply:
Kept as .ts in order to successfully generate types (generating .d.ts from .js was providing really strange results...). See PR comment on how we now generate .js files though...